website/siteConfig.js

```diff
@@ -11,20 +11,20 @@
 // List of projects/orgs using your project for the users page.
 const users = [
   {
-    caption: 'User1',
+    caption: 'WePay',
     // You will need to prepend the image path with your baseUrl
     // if it is not '/', like: '/test-site/img/image.jpg'.
-    image: '/img/undraw_open_source.svg',
-    infoLink: 'https://www.facebook.com',
+    image: '/waltz/img/we-pay-logo-color.svg',
+    infoLink: 'https://go.wepay.com/about-wepay/',
     pinned: true,
   },
 ];
 
 const siteConfig = {
   title: 'Waltz', // Title for your website.
   tagline: 'An open source quorum-based distributed write-ahead log',
-  url: 'https://your-docusaurus-test-site.com', // Your website URL
-  baseUrl: '/', // Base URL for your project */
+  url: 'https:wepay.github.io', // Your website URL
```

Comment:
Should this have `://` ?

Reply:
It seems like it is optional.
The succeeding example didn't have an ending `/`, so didn't put it here.

Reply:
Oops.. shoot! I got it! Will update.